queries.sql

```diff
@@ -124,3 +124,83 @@ HAVING MAX(newcount.count) =
       FROM animals
       JOIN owners ON animals.owner_id = owners.id
       GROUP BY owners.full_name) AS newcount);
+
+-- Day 4
+
+SELECT a.date, b.name as pokemon, c."name" as vet 
+FROM visits a 
+JOIN animals b ON a.animal_id = b.id 
+JOIN vets c ON a.vet_id = c.id
+where c."name" = 'William Tatcher'
+order by a."date" desc limit 1;
```

Comment:
[Optional]
- [ ] You should use better aliases and names. Using just a letter makes it hard to read and track. Imagine yourself working on a big project and trying to figure out things, which one do you prefer? Please compare the example code with your code block.

```sql
SELECT animals.name, visits.visit_date
FROM visits
LEFT JOIN animals ON animals.id = visits.animal_id
LEFT JOIN vets ON vets.id = visits.vet_id
WHERE vets.name = 'William Tatcher'
ORDER BY visits.visit_date DESC
LIMIT 1;
```

```sql
SELECT a.date, b.name as pokemon, c."name" as vet 
FROM visits a 
JOIN animals b ON a.animal_id = b.id 
JOIN vets c ON a.vet_id = c.id
where c."name" = 'William Tatcher'
order by a."date" desc limit 1;
```

Reply:
You are right, thank you. @sinansevgi